### ISSUE TITLE
Could not invoke thaumcraft.api but I don't have thaumcraft

### ISSUE THREAD

**Issue by AnnexedNovaSpace** [Azanor/thaumcraft#1797]
I host a modded server on my computer. For some reason, it gives me the same exact error in the server log several times that says "[Server thread/WARN] [FML]: [Thaumcraft API] Could not invoke thaumcraft.common.lib.crafting.ThaumcraftCraftingManager method getObjectTags" but I don't have Thaumcraft. I never had Thaumcraft installed on my server, or my client. Why is my server trying to do this when I never had Thaumcraft installed? Does one of my mods use Thaumcraft's API or something? I'm so confused...

**Comment by T4ge** [comment]
Because something you have installed have Thaumcraft integration I assume

**Comment by AnnexedNovaSpace** [comment]
Hmm... Maybe... But I never downloaded anything to integrate any of my mods with Thaumcraft. Do some mods come with it built-in?

**Comment by T4ge** [comment]
Yes quite a few does, even fully standalone mods like botania and Tinkers
Construct have it.
